### ISSUE TITLE
Retrying setup: Start Listening was not completed in timely manner

### ISSUE THREAD

**Issue by jouster1974** [fuatakgun/eufy_security#352]
started seeing this in the last day or so and again the floodlight camera feed is once again gone.

Multiple restarts don't fix

Currently on 2022.5.0   2022.05.3

OS 8.1

All other cameras work fine apart from floodlight last event feed but this error is always in palce

**Comment by JasSmiths** [comment]
+1 I've got this issue as well but causing me not to be able to setup anything as only just installed

**Comment by dsutherlin** [comment]
I am also having the same issue with the error message below:

Retrying setup: Start Listening was not completed in timely manner

**Comment by joske500** [comment]
+1 installed and configured everything as specified, still got this error

**Comment by SuNzZeR** [comment]
I have the same issue after Update Eufy Security Add-on from 0.8.4 to 0.9.1

`Setup erneut versuchen: Start Listening was not completed in timely manner`

Eufy Security Add-on Log:
`2022-06-03 10:30:40.245  INFO  Eufy Security server listening on host 0.0.0.0, port 3000 `
`2022-06-03 10:31:02.323  INFO  Client disconnected with ip: 172.30.32.1 port: 50334 code: 1000 reason: ` 
`2022-06-03 10:31:07.575  INFO  Client disconnected with ip: 172.30.32.1 port: 50340 code: 1000 reason:`  
`2022-06-03 10:31:17.839  INFO  Client disconnected with ip: 172.30.32.1 port: 50344 code: 1000 reason:`

After Reinstall I have the same issue.

**Comment by robconway** [comment]
Same issue, no devices found and listening error

**Comment by Noble-One** [comment]
Current version: 0.9.1
Home Assistant Core 2022.6.2
Home Assistant Supervisor 2022.05.3
Home Assistant OS 8.1

I am seeing the same issues.  I have removed and added a few times now.

Integrations shows an error:
Retrying setup: Start Listening was not completed in timely manner

Eufy Security Add-on Log:
2022-06-06 00:46:05.160  INFO  Eufy Security server listening on host 0.0.0.0, port 3000 
2022-06-06 00:47:13.362  INFO  Client disconnected with ip: 172.30.32.1 port: 55766 code: 1000 reason:  
2022-06-06 00:48:34.127  INFO  Client disconnected with ip: 172.30.32.1 port: 42502 code: 1000 reason:  
2022-06-06 00:49:54.896  INFO  Client disconnected with ip: 172.30.32.1 port: 45896 code: 1000 reason:

**Comment by Gtt1229** [comment]
I didn't have this issue, but after a reinstall, I am also having the same results.

**Comment by nkback** [comment]
Also having this issue.

**Comment by jellejans** [comment]
@fuatakgun same issue here, are we missing something ?

**Comment by mikekeil-wf** [comment]
Same issue here.

**Comment by stijnbamps** [comment]
I'm also having this issue.

**Comment by advicepyro** [comment]
I encountered this issue and have a workaround of sorts.

I ran into this while trying (in vain) to configure a systemd service for running the eufy-security-ws server. I have a HA Core install so I'm manually starting and stopping the Docker container. In the process, I must've made too many login attempts on that account, so the account is placed into some timeout where it requires the captcha to start up.

For the account I assigned to use with HA, when starting up the eufy-security-ws server all I see in the container logs is:

```
2022-06-14 00:23:08.350  INFO  Eufy Security server listening on host 0.0.0.0, port 3000
2022-06-14 00:23:22.344  INFO  Client disconnected with ip: 172.17.0.1 port: 49848 code: 1000 reason:
2022-06-14 00:23:27.322  INFO  Client disconnected with ip: 172.17.0.1 port: 49850 code: 1000 reason:
2022-06-14 00:23:37.397  INFO  Client disconnected with ip: 172.17.0.1 port: 49854 code: 1000 reason:
```

and when I tried logging into that account via the tablet app, I get a captcha.

So I switched to another account (but still linked to the same home) which I knew does not _currently_ present the captcha challenge, lo and behold when I started eufy-security-ws:

```
2022-06-14 00:33:47.700  INFO  Eufy Security server listening on host 0.0.0.0, port 3000
2022-06-14 00:33:54.310  INFO  Connected to station TXXXXXXXXXXXXX on host 192.168.XX.XX and port 22148
2022-06-14 00:33:55.556  INFO  Push notification connection successfully established
```

When running HA as supervised, you can enter the captcha there, but I don't know of how to get around the captcha while running the container manually.

tl;dr workaround:
- wait a few hours to pass before you don't need the captcha to login
- switch to another linked account that doesn't ask you for a captcha

**Comment by fuatakgun** [comment]
can someone re-rest with latest add-on version? please restart your home assistant after update.

**Comment by alanmilinovic** [comment]
Still the same issue.

**Comment by robconway** [comment]
@alwin006 thanks for the tip but unfortunately that didn't work for my installation. 

Very buggy and frustrating!

**Comment by olivercastle** [comment]
I also have the same issue. 

Using a different account with shared devices did resolve the issue for me also as a workaround.
